feeder/feeder.py

```diff
@@ -300,7 +302,7 @@ def validate_production(obj, country_code):
         raise Exception("Data from %s can't be in the future" % country_code)
     if obj.get('production', {}).get('unknown', None) is None and \
         obj.get('production', {}).get('coal', None) is None and \
-        country_code not in ['CH', 'NO']:
+        country_code not in ['CH', 'NO', 'IS']:
```

Comment:
The purpose of this line is to make exceptions for specific countries that we want to show on the map even though they have no coal data.
However, having no coal data is different from having coal set to 0. For IS, if we can confirm that they do not produce any coal, we should set the `coal` key to 0 in the parser. We should do the same for all production types that we know IS doesn't have.